mkdocs.yml

```diff
@@ -57,6 +57,8 @@ nav:
     - Configuration: reference/configuration.md
     - REST API: reference/rest_api.md
     - Metadata: reference/metadata.md
+  - Maintenance:
```

Comment:
Minor: Should we call it "Developers guide" or "Maintainers guide" instead?

Reply:
Btw, let's just discuss on Friday's morning standup.

Reply:
Decided in standup: "Maintainers guide"

Reply:
changed :)